src/Components/Authorization/src/AuthorizeRouteView.cs

```diff
@@ -54,6 +54,12 @@ public AuthorizeRouteView()
         [Parameter]
         public RenderFragment Authorizing { get; set; }
 
+        /// <summary>
+        /// The resource to which access is being controlled.
+        /// </summary>
+        [Parameter]
+        public object Resource { get; set; }
```

Comment:
Nice! You'll probably need to regenerate the reference assemblies for this.

You can do that by running `dotnet msbuild /t:GenerateReferenceSource` in the `src/Components/Authorization` directory.

Reply:
I've regenerated from `src/Components/Authorization`